website/blog/2023-04-18-aluminium.md

```diff
@@ -0,0 +1,391 @@
+    <td align="center">100</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">80</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">20</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
+  </tr>
+</tbody>
+</table>
+
+For full details: [https://github.com/scalameta/metals/milestone/56?closed=1]
+(https://github.com/scalameta/metals/milestone/56?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Error Reports](#introduce-error-reports)
+- [Code action for converting dependencies from sbt to mill style](#code-action-for-converting-dependencies-from-sbt-to-mill-style)
+- [Improvements to semantic highlighting](#improvements-to-semantic-highlighting)
+- [Miscellaneous](#miscellaneous)
+
+## Error Reports
+
+Starting with this release, metals will automatically generate error reports, which can later be used by users to attach to github issue for additional information. Upon any error or incorrect state metals will create an error report in `.metals/.reports` directory within the workspace. All the reports have their paths anonymised.
+
+Currently, we create two types of reports:
+
+- _incognito_ under `.metals/.reports/metals`: those error reports contain only metals related stacktraces and should not contain any sensitive/private information
+- _unsanitized_ under `.metals/.reports/metals-full`: those error reports may contain come sensitive/private information (e.g. code snippets)
+
+To make it easier to attach reports to github issues the `zip-reports` command will zip all the **incognito** reports into `.metals/.reports/report.zip`, while unsanitized reports will have to be browsed and attached by hand. In order not to clutter the workspace too much, only up to 30 last reports of each kind are kept at a time.
+
+When running into an problem in VSCode users can use the `Metals: Open new github issue` command. This will create a template for issue with all basic info such as Metals version, used build server etc.. Next, you can browse through reports and drag and drop chosen ones to your GitHub issue. Invoking `Metals: Zip reports` command will create a zip of all the incognito reports, that you may attach too.
+
+![Reports](https://i.imgur.com/wBwFjpZ.gif)
+![Zip reports](https://i.imgur.com/YN3U3N9.gif)
+
+## Code action for converting dependencies from sbt to mill style
+
+New code action for scala-cli `using (dep | lib | plugin)` directives, which allows to convert dependencies from sbt style to mill style.
```

Comment:
This actually confuses me a bit. If this code actions is meant to convert sbt style dependencies to be used via using  directives I wouldn't actually call it "mill style". I think users might get confused here and when they see "mill style" they may be thinking the `$ivy.` style, which isn't true here. I'd maybe recommend rephrasing this to say that it turns sbt style dependencies into scala-cli complaint ones or something like that. _Unless_ this also works anywhere and not just in using directives. If that's the case then we should also mention that.